packages/core/src/routes/swagger/index.ts

```diff
@@ -71,6 +73,7 @@ const buildOperation = (
   const queryParameters = [
     ...buildParameters(query, 'query'),
     ...(hasPagination ? paginationParameters : []),
+    ...(path === '/users' && method === 'get' ? [searchParameters] : []),
```

Comment:
if i remembered correctly, `/applications` also have this option for advanced searching. @mostafa would you like to double check this? ty

Reply:
Sure.

Reply:
you are a hero!